library/db/src/main/java/org/keynote/godtools/android/db/GlobalActivityAnalyticsMapper.java

```diff
@@ -0,0 +1,57 @@
+package org.keynote.godtools.android.db;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+
+import org.cru.godtools.model.GlobalActivityAnalytics;
+
+import androidx.annotation.NonNull;
+
+import static org.keynote.godtools.android.db.Contract.GlobalActivityAnalyticsTable.COLUMN_COUNTRIES;
+import static org.keynote.godtools.android.db.Contract.GlobalActivityAnalyticsTable.COLUMN_GOSPEL_PRESENTATIONS;
+import static org.keynote.godtools.android.db.Contract.GlobalActivityAnalyticsTable.COLUMN_LAUNCHES;
+import static org.keynote.godtools.android.db.Contract.GlobalActivityAnalyticsTable.COLUMN_USERS;
+
+public class GlobalActivityAnalyticsMapper extends BaseMapper<GlobalActivityAnalytics> {
```

Comment:
Implement this in Kotlin, it will be much simpler. There should be a few Mappers already converted to Kotlin if you need examples